yt_dlp/extractor/playsuisse.py

```diff
@@ -134,12 +135,35 @@ class PlaySuisseIE(InfoExtractor):
             id
             url
         }'''
+    _ID_TOKEN = None
```

Comment:
since we reuse these values multiple times it will be more readable/maintainable to have them as class vars
```suggestion
    _LOGIN_BASE_URL = 'https://login.srgssr.ch/srgssrlogin.onmicrosoft.com'
    _LOGIN_PATH = 'B2C_1A__SignInV2'
    _ID_TOKEN = None
```